controller/schema/schema.go

```diff
@@ -39,6 +39,7 @@ func Load(schemaRoot string) error {
 		schema := &jsonschema.Schema{Cache: schemaCache}
 		err = schema.ParseWithoutRefs(file)
 		if err != nil {
+			file.Close()
```

Comment:
I believe you can move this up one line outside of the `if` and then delete the close below.